Google Cloud/google-folder/folders.yaml

```diff
@@ -0,0 +1,6 @@
+components: 
+  parent_id: "folders/227414018885"
```

Comment:
Follow the same pattern of structure as other adapters https://github.com/mesoform/Multi-Cloud-Platform-Services-1/blob/main/docs/GCP_CLOUDRUN.md